backend/src/app/http/middleware.clj

```diff
@@ -34,11 +31,12 @@
   {:name ::params
    :compile (constantly ymw/wrap-params)})
 
-(def ^:private json-mapper
-  (json/mapper
-   {:encode-key-fn str/camel
-    :decode-key-fn (comp keyword str/kebab)
```

Comment:
@niwinz I think the keys transformation must be kept. Otherwise it says as `is-owner` which is harder to go through in JS (`data['is-owner']` instead of `data.isOwner`)

Reply:
oh, you are right, i will fix it